gtdynamics/kinematics/KinematicsSlice.cpp

```diff
@@ -105,11 +137,22 @@ Values Kinematics::initialValues<Slice>(const Slice& slice, const Robot& robot,
 
 template <>
 Values Kinematics::inverse<Slice>(const Slice& slice, const Robot& robot,
-                                  const ContactGoals& contact_goals) const {
-  auto graph = this->graph(slice, robot);
+                                  const ContactGoals& contact_goals,
+                                  bool contact_goals_as_constraints) const {
```

Comment:
Why have this flag? Comment on this in PR comment,

Reply:
done